### ISSUE TITLE
K2 Items no longer saving in Joomla 3.7

### ISSUE THREAD

**Issue by Simon-Davies** [joomla/joomla-cms#15809]
### Steps to reproduce the issue

Edit a really long blog post and click Save

### Expected result

Item saves successfully

### Actual result

Redirects to blank page with URL: www.domain.com/administrator/index.php?

### System information (as much as possible)

PHP 7.0

Joomla 3.7

K2 (latest github build)

Admin Tools Pro (latest version)

### Additional comments

This has only just started happening since upgrading from the last version of Joomla to Joomla 3.7

**Comment by ssnobben** [comment]
Is this a Joomla problem or a K2 problem? Have you ask K2 developer?<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/15809">issues.joomla.org/tracker/joomla-cms/15809</a>.</sub>

**Comment by Simon-Davies** [comment]
The latest version of K2 worked perfectly with the last version of Joomla.

K2 hasn't been updated since then so it has to be anbissue introduced by Joomla 3.7

**Comment by joomdonation** [comment]
Maybe it is related to this issue https://github.com/joomla/joomla-cms/issues/15673 . The solution for now is restore the file libraries/joomla/filter/input.php from Joomla 3.6.5 version as mentioned by other users

**Comment by Simon-Davies** [comment]
Thank you. I'll give it a shot :)

**Comment by AyelGoth** [comment]
After updating to Joomla 3.7, the K2 component does not allow me to publish articles. When you try to save the article, it takes too long and a blank screen appears.

In the error reports I get the following message:

Fatal error: Maximum execution time of 90 seconds exceeded in /var/www/html/libraries/vendor/joomla/string/src/phputf8/mbstring/core.php on line 41

I did a test, I created a publication that contained no accents, no quotes. The item was saved correctly. I created another article that contained special characters, quotes, accents, I can not keep. I think the problem has something to do with UTF8.
